upstairs/src/guest.rs

```diff
@@ -342,9 +343,13 @@ impl Guest {
         rx.wait().await
     }
 
-    pub async fn query_extent_size(&self) -> Result<Block, CrucibleError> {
-        self.send_and_wait(|done| BlockOp::QueryExtentSize { done })
-            .await
+    pub async fn query_extent_info(
```

Comment:
Why is this implemented twice (once in `impl Guest` and once in `impl BlockIO for Guest`)?

Reply:
I don't know why. Maybe an artifact of existing before BlockIO trait?
`activate_with_gen()` and `query_work_queue()` are also doubled up.  I'll remove the duplicates as
I don't see any differences that we need.